.github/workflows/build.yml

```diff
@@ -299,13 +131,11 @@ jobs:
       - build
     strategy:
       matrix:
-        arch: ["amd64"]
+        arch: ["amd64"] #, "386", "arm", "arm64"]
```

Comment:
Is there a reason we only build & push `amd64`? Just confirming that this is the only docker arch we'd like to support.

Reply:
This was just because we were previously using this workflow in a temporary capacity to "pre-check" our release branches using a similar set of steps as the release process would follow, and the legacy release process only generates amd64 images. If CRT is ready to publish images for other architectures then this seems like a good time to enable them, since folks have been requesting it for a while.

Reply:
Ok we'll enable them !